rllib/tuned_examples/ppo/multi_agent_pendulum_ppo.py

```diff
@@ -8,10 +8,16 @@
 config = (
     PPOConfig()
     .environment("multi_agent_pendulum")
-    .rollouts(num_envs_per_worker=10, batch_mode="complete_episodes")
+    .rollouts(
+        num_envs_per_worker=20,
+        observation_filter="MeanStdFilter",
+        num_rollout_workers=0,
+        batch_mode="complete_episodes",
+    )
     .training(
-        train_batch_size=2048,
+        train_batch_size=512,
         lambda_=0.1,
+        gamma=0.95,
```

Comment:
These are all parameters taken from rllib/tuned_examples/ppo/pendulum-ppo.yaml.
The reasoning here is ....
1. Although this is a multi-agent test, it has only one agent and so the whole setting is extremely similar.
2. Both tests have the same length on CI, but rllib/tuned_examples/ppo/multi_agent_pendulum_ppo.py times out without these changes.